specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/sourceControlSyncJobStreams.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Please add a reference to this file in the input files [here](https://github.com/MiaRomero/azure-rest-api-specs/blob/ca7362df951048d68f8e1b46995b7d43a3b92c1d/specification/automation/resource-manager/readme.md)

Reply:
Added a reference in the readme file.